src/wasm/hostcall/bindings.json

```diff
@@ -0,0 +1,50 @@
+{
+  "env": {
+      "emit": "emit",
+      "raise": "raise",
+      "register": "register"
+  },
+  "wasi_snapshot_preview1": {
```

Comment:
This can be removed to use the `Bindings` more directly: https://docs.rs/lucet-module/0.6.1/lucet_module/bindings/struct.Bindings.html